addons/zeus/functions/fnc_moduleArtillery.sqf

```diff
@@ -15,71 +15,74 @@
  *
  */
 #include "script_component.hpp"
-
+//CHECK even neccessary?
```

Comment:
```suggestion
```

Reply:
Is the module called locally on the zeus?
If so then you can remove the checks.

Reply:
As of the documentation:
The module function is executed in the unscheduled environment on the client that placed the module.